controllers/topics.controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectTopics } =require('../models/topics.model')
+
+exports.getTopics = (req, res, next) => {
+    selectTopics(req.query)
+    .then((topics) => {
+        
+        res.status(200).send({ topics })
+    }).catch(next)
+}
```

Comment:
This is a well named function name for the controller, well done.

Is there any reason why you are passing `req.query` into `selectTopics`? The API endpoint doesn't accept any request parameters, and also you don't appearing to be accepting any arguments in the model for the `selectTopics` function. 

Sending the data back on a key of `topics` is appropriate and correct, well done.

And well done for adding the `.catch` block to handle any errors.